packages/web-app-files/src/mixins/actions/rename.ts

```diff
@@ -41,6 +41,12 @@ export default {
               return false
             }
 
+            // FIXME: Remove this check as soon as renaming shares works as expected
```

Comment:
Could you add the ocis issue link here? Otherwise it's really hard to find out when it's supposed to work again.